routes/api.js

```diff
@@ -7,6 +7,14 @@ var debug = require('debug')('dashboard-proxy:server');
 var error = require('debug')('dashboard-proxy:server:error');
 var Buffer = require('buffer').Buffer;
 var BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+var BufferList;
+try {
+    // from source tree
+    BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+} catch(e) {
+    // from install location
+    BufferList = require('../../websocket/vendor/FastBufferList');
+}
```

Comment:
Oh! The error is because you still have the older require code at line 9.

I'm looking at `websocket` to see if we can do something about this include. Don't like it.